test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -4187,6 +4190,27 @@ def tracejob_match(self, msg, id=None, n=50, tail=True,
                                regexp, max_attempts, interval, starttime,
                                endtime, level, existence)
 
+    def del_hooks_nodes_qs(self, del_hooks=True,
+                           del_nodes=True,
+                           del_qs=True,
+                           del_vdef=True):
+        """
+        Delete nodes, queues and hooks
+        """
+        if del_hooks:
+            self.delete_hooks()
+        if del_vdef or del_nodes:
+            self.cleanup_reservations()
```

Comment:
since you also delete reservations here, please add that to the method name and docstring

Reply:
or you can call the method something generic, like "delete_current_state", up to you. But do mention what all will get deleted in the docstring

Reply:
Added docstring & changed function name